WTWSMS/common/religions/00_religions.txt

```diff
@@ -4936,11 +4975,13 @@ indian_group = {
 		intermarry = buddhist
 	}
 	mimamsa = {
+		has_coa_on_barony_only = yes
+
```

Comment:
Many of the symbols represent Indian gods and beliefs that did not exist among the Mimamsas, who reject most Hindu deities and focus on the person and the clergy. Also, the entire "replace COA" thing always sucked anyway, but I am not in a place to do that.